docs/other_vscode.md

```diff
@@ -0,0 +1,117 @@
+
+The purpose of the is page is to document how to set-up VS Code for developing QMK Firmware.
+
+This guide covers how to configure everything needed on Windows and Ubuntu 18.04
+
+# Set up VS Code
+Before starting, you will want to make sure that you have all of the build tools set up, and QMK Firmware cloned. Head to the the [Newbs Getting Started Guide](newbs_getting_started.md) to get things set up, if you haven't already.
+
+## Windows
+
+### Prerequisites
+
+* [Git for Windows](https://git-scm.com/download/win) (This link will prompt to save/run the installer)
+  
+  1. Disable all of the options but `Git LFS (Large File Support)` and `Check daily for Git for Windows updates`. 
+  2. Set the default editor to `Use Visual Studio Code as Git's default editor`
+  3. Select the `Use Git from Git Bash only` option, since that's the option that you should use here.
+  4. For the `Choosing HTTPS transport backend`, either option should be fine.
+  5. Select the `Checkout as-is, commit Unix-style line endings` option. QMK Firmware uses Unix style commits.
+  6. For the extra options, leave the default options as is. 
+
+  This software is needed for Git support in VS Code. It may be possible to not include this, but it is much simpler to just use this. 
+
+* [Git Credential Manager for Windows](https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases) (Optional) 
+
+  This software provides better support for Git by providing secure storage for git credentials, MFA and personal access token generation. 
+  
+  This isn't strictly needed, but we would recommend it. 
+
+
+### Installing VS Code
+
+1. Head to [VS Code](https://code.visualstudio.com/) and download the installer
+2. Run the installer
+
+This part is super simple.  However, there is some configuration that we need to do to make things are configured correctly.
+
+### Configuring VS Code
+
+First, we need to set up IntelliSense. This isn't strictly required, but it will make your life a LOT easier. To do this, we need to create the `.vscode/c_cpp_properies.json` file in the QMK Firmware folder, You can do this all manually, but I've done most of the work already. 
+
+Grab [this file](https://gist.github.com/drashna/48e2c49ce877be592a1650f91f8473e8) and save it.  You may need to edit this file, if you didn't install MSYS2 to the default location, or are using WSL/LxSS.  
+
+Once you have saved this file, you will need to reload VS Code, if it was already running. 
+
+?> You should see an `extensions.json` and `settings.json` file in the `.vscode` folder, as well.
+
+
+Now, we will set up the MSYS2 window to show up in VSCode as the integrated terminal.  This has a number of advantages. Mostly, you can control+click on errors and jump to those files.  This makes debugging much easier.  It's also nice, in that you don't have to jump to another window. 
+
+1. Click <kbd><kbd>File</kbd> > <kbd>Preferences ></kbd> > <kbd>Preferences</kbd> </kbd>
```

Comment:
That should probably read:
File > Preferences > Settings

Reply:
haha, yeah, you're right! 

(next time, you can hit "ctrl+g" to create a "suggestion" on git, and that can be directly applied here)

```suggestion
1. Click <kbd><kbd>File</kbd> > <kbd>Preferences ></kbd> > <kbd>Settings</kbd> </kbd>
```